.vscode/settings.json

```diff
@@ -4,5 +4,6 @@
   "eslint.workingDirectories": ["./website", "./docs"],
   "python.testing.pytestArgs": ["backend/tests", "oasst-shared/tests"],
   "python.testing.unittestEnabled": false,
-  "python.testing.pytestEnabled": true
+  "python.testing.pytestEnabled": true,
+  "editor.tabSize": 2
```

Comment:
if you could remove the changes in this file we can merge this directly

Reply:
Done